.circleci/config.yml

```diff
@@ -64,6 +64,7 @@ executors:
       - image: mcr.microsoft.com/playwright:v1.31.0-focal
     working_directory: *playwright_path
     environment:
+      TZ: "America/New_York"
```

Comment:
On CircleCi, test verification of date (mm/dd/yyyy) after 9 PM fails due to UTC timezone on CircleCI VM.